cfpb/consumerfinance.gov
# P4C: bypass grad path tool's intro screen for returning users

Returning users (determined for this purpose by the presence of an `iped` value in the url) should go directly to grad path app instead of viewing the intro screen. 

## Additions

- Returning users check
- Loading notification conditionally displayed instead of intro content for returning users

## Changes

- Automatically start app rather than hooking up get started button handler if iped is detected in URL


## How to test this PR

1. Check out branch and run gulp
2. Navigate to [grad path tool with iped in url](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=106397) and verify that app starts automatically and a loading screen displays instead of the intro content while the app launches
3. Navigate to [grad path tool without an iped value](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/) and verify that intro screen is still displayed and app is still launched by get started button


## Screenshots
<img width="1463" alt="Screen Shot 2021-04-12 at 9 45 51 AM" src="https://user-images.githubusercontent.com/778171/114895340-49aa9900-9dc4-11eb-8b1b-6a175d1cff86.png">


## Notes and todos

- There may be better ways to handle this page branching (and loading screen messaging), but this is a quick MVP for user testing


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
